### ISSUE TITLE
Rule to minimum number of lines?

### ISSUE THREAD

**Issue by MatanGbr** [sindresorhus/eslint-plugin-unicorn#1695]
### Description

I know it sounds weirds, but I believe there is no place for really short JavaScript file, and that these tiny bits of code should be combined with other tiny bits of code to make a file of at least 5 lines.
I have tried to play with ESLint's default [max-lines](https://github.com/eslint/eslint/blob/main/lib/rules/max-lines.js) without much success (it always broke all the other problems from showing) so I decided to start with a very basic version, but with no luck yet:

```js
'use strict'
module.exports = {
  meta: {
    type: 'suggestion',
    schema: [],
  },
  create(context) {
    const sourceCode = context.getSourceCode();
    let lines = sourceCode.lines.map((text, i) => ({
      lineNumber: i + 1,
      text
    }));
    return {
      "Program:exit"() {
        if (lines.length < 5) {
          context.report({
            node: node.property,
            message: 'File should have more lines',
          })
        }
      },
    }
  },
}
```

Any help would be welcomed!

### Fail

```js
var x= 'me';
alert(x)
```
```js
var y= 'you';
alert(y)
```

### Pass

```js
var x= 'me';
alert(x)
var y= 'you';
alert(y)
```

**Comment by fisker** [comment]
I think this is a prefect module.

```js
// utils/noop.js

export default () => {};
```

**Comment by MatanGbr** [comment]
You right. But it still doesn't work 😞

**Comment by fisker** [comment]
> But it still doesn't work 

What do you mean? You mean your rule doesn't work?

```diff
-             node: node.property,
-             node,
```

**Comment by MatanGbr** [comment]
Yeah, still got nothing when I test it on a file with less than 10 lines:

> 
>         "special/min-lines": "error"
> 
> 

```
'use strict'
export default () => module.exports = {
  meta: {
    type: 'suggestion',
    schema: [],
  },
  create(context) {
    const sourceCode = context.getSourceCode();
    let lines = sourceCode.lines.map((text, i) => ({
      lineNumber: i + 1,
      text
    }));
    return {
      "Program:exit"() {
        if (lines.length < 5) {
          context.report({
            node: node,
            message: 'File should have more lines',
          })
        }
      },
    }
  },
}


```

You might have an idea? 🙏

**Comment by fisker** [comment]
This is not the place you ask for how to write rules(check ESLint docs and there are many examples here).

If you want proposal a rule like this please follow the issue template.
